srv/salt/ceph/ganesha/files/silver-common.conf.j2

```diff
@@ -6,9 +6,9 @@ EXPORT
 {
 	Export_ID={{ loop.index }};
 
-	Path = "/";
+	Path = "demo";
```

Comment:
nope, s3 uses a global bucket namespace instead of a tenant namespace by default